src/App.js

```diff
@@ -35,14 +36,26 @@ class App extends React.Component {
     this.state = {
       todoItems,
       //Q: why is this kept here? it isn't a slice of state we're tracking, is it? Does this enable the rerendering when it changes???
-      toggleCompleted: () => {},
-      removeCompleted: () => {},
+      //Yetti says These are definitely weird here & I didn't need them.
```

Comment:
You have no idea how much this was bugging me from lecture. Couldn't figure out why we were tracking state of functions.